### PR TITLE
my-bike

### PR DESCRIPTION
**Pull request by VolodymyrKirichenko**
- [DEMO LINK](https://VolodymyrKirichenko.github.io/layout_my-bike/)

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -20,7 +20,11 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
     - run: npm install
+<<<<<<< Updated upstream
     - run: npm start & sleep 5 && npm test
+=======
+    - run: npm test
+>>>>>>> Stashed changes
```
Comment (SerhiiKirik) [inline .github/workflows/test.yml]: why you add this?

File: .linthtmlrc.json [hunk 2]
```diff
@@ -1,3 +1,49 @@
...
+  "tag-bans": [
+    "b",
+    "i",
+    "u",
+    "center",
+    "style",
+    "marquee",
+    "font",
+    "s"
+  ],
```
Comment (SerhiiKirik) [inline .linthtmlrc.json:38]: and this?

File: src/styles/blocks/header.scss [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+        transform: translate(-100%, -50%);
+        transition-duration: 0.3s;
+
+        display: block;
+        width: max-content;
+        padding-right: 20px;
+        font-weight: bold;
+        font-size: 14px;
+        line-height: 100%;
```
Comment (SerhiiKirik) [inline src/styles/blocks/header.scss]: Change this to px in all cases

File: src/styles/blocks/about.scss [hunk 4]
```diff
@@ -0,0 +1,37 @@
+.about {
+  &__title {
+    margin: 0 0 32px;
+
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 100%;
+
+    letter-spacing: -1.5px;
```
Comment (SerhiiKirik) [inline src/styles/blocks/about.scss]: you use this 5 times, you can create variables for this

File: src/styles/blocks/about.scss [hunk 5]
```diff
@@ -0,0 +1,37 @@
+.about {
+  &__title {
+    margin: 0 0 32px;
+
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 100%;
+
```
Comment (SerhiiKirik) [inline src/styles/blocks/about.scss]: mixin for this

File: src/styles/blocks/about.scss [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+      font-size: 64px;
+    }
+  }
+
+  &__text {
+    display: block;
+
+    font-weight: 500;
+    font-size: 23px;
+    line-height: 140%;
```
Comment (SerhiiKirik) [inline src/styles/blocks/about.scss]: px

File: src/styles/blocks/about.scss [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+      font-size: 48px;
+    }
+
+    @include onDesktop {
+      font-size: 64px;
+    }
+  }
+
+  &__text {
+    display: block;
```
Comment (SerhiiKirik) [inline src/styles/blocks/about.scss]: p tag has display: block by defauld

File: src/styles/blocks/compare.scss [hunk 8]
```diff
@@ -0,0 +1,23 @@
+.compare {
+  &__title {
+    text-align: center;
+
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 32px;
```
Comment (SerhiiKirik) [inline src/styles/blocks/compare.scss]: use this text mixin here

File: src/styles/blocks/compare.scss [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+  &__title {
+    text-align: center;
+
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 32px;
+
+    text-align: center;
+    letter-spacing: -1.5px;
+    text-transform: capitalize;
```
Comment (SerhiiKirik) [inline src/styles/blocks/compare.scss]: DRY - you use this 2 times

File: src/styles/blocks/contact-us.scss [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+
+  &__info-link {
+    color: inherit;
+    text-decoration: none;
+
+    transition: border-color 0.3s;
+
+    &:hover {
+      color: #fff;
+      text-shadow: 0 0 10px #fff;
```
Comment (SerhiiKirik) [inline src/styles/blocks/contact-us.scss]:
```suggestion
      color: $colorWhite;
      text-shadow: 0 0 10px $colorWhite;

```

File: src/styles/blocks/contact-us.scss [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+    }
+  }
+
+  &__info-title {
+    margin: 0 0 8px;
+
+    font-size: 14px;
+    line-height: 27px;
+
+    color: #fff;
```
Comment (SerhiiKirik) [inline src/styles/blocks/contact-us.scss]:
```suggestion
    color: $colorWhite;

```

File: src/styles/blocks/form.scss [hunk 12]
```diff
@@ -0,0 +1,27 @@
+.form {
+  &__field {
+    display: block;
```
Comment (SerhiiKirik) [inline src/styles/blocks/form.scss]: why block?

File: src/styles/blocks/form.scss [hunk 13]
```diff
@@ -0,0 +1,27 @@
...
+
+    padding: 14px 16px;
+
+    background: #f8f8fa;
+    border: 1px solid transparent;
+    border-radius: 40px;
+
+    &:hover {
+      background: #f8f8fa;
+      border: 1px solid #d5d7de;
```
Comment (SerhiiKirik) [inline src/styles/blocks/form.scss]:
```suggestion
    background: $variables;
    border: 1px solid transparent;
    border-radius: 40px;

    &:hover {
      background: $variables;
      border: 1px solid $variables;

```

File: src/styles/blocks/grid.scss [hunk 14]
```diff
@@ -0,0 +1,52 @@
...
+  @include onTablet {
+    grid-template-columns: repeat(6, 1fr);
+    column-gap: 24px;
+  }
+
+  @include onDesktop {
+    grid-template-columns: repeat(12, 1fr);
+    column-gap: 24px;
+  }
+
```
Comment (SerhiiKirik) [inline src/styles/blocks/grid.scss:17]:
```suggestion
```

File: src/styles/blocks/header.scss [hunk 15]
```diff
@@ -0,0 +1,106 @@
...
+    background-image: url(../images/photos/phone.svg);
+
+    @include onTablet {
+      &::before {
+        content: "+1 234 5555-55-55";
+        position: relative;
+        top: 50%;
+
+        transform: translate(-100%, -50%);
+        transition-duration: 0.3s;
```
Comment (SerhiiKirik) [inline src/styles/blocks/header.scss]: better will be if you use transition: opacity 0.3s

File: src/styles/blocks/header.scss [hunk 16]
```diff
@@ -0,0 +1,106 @@
...
+      &:hover::before {
+        opacity: 1;
+        pointer-events: all;
+      }
+    }
+  }
+
+  &__burger-menu {
+    background-image: url(../images/photos/burger.svg);
+    transition-duration: 0.3s;
```
Comment (SerhiiKirik) [inline src/styles/blocks/header.scss]: the same

File: src/styles/blocks/header.scss [hunk 17]
```diff
@@ -0,0 +1,106 @@
...
+
+    &:hover {
+      background-image: url(../images/photos/burger-hover.svg);
+    }
+  }
+
+  &__title {
+    font-family: Poppins, sans-serif;
```
Comment (SerhiiKirik) [inline src/styles/blocks/header.scss]:
```suggestion
```

File: src/styles/blocks/menu.scss [hunk 18]
```diff
@@ -0,0 +1,116 @@
+.menu {
+  width: 100%;
+
+  overflow: auto;
+
+  background-color: #c2c2c2;
```
Comment (SerhiiKirik) [inline src/styles/blocks/menu.scss]:
```suggestion
  background-color: $variables;

```

File: src/styles/blocks/menu.scss [hunk 19]
```diff
@@ -0,0 +1,116 @@
...
+
+  background-color: #c2c2c2;
+
+  &__container {
+    max-width: 1260px;
+    width: 100%;
+    margin: 0 auto;
+
+    height: 100vh;
+    padding: 29px 20px 20px;
```
Comment (SerhiiKirik) [inline src/styles/blocks/menu.scss]:
```suggestion
    max-width: 1260px;
    width: 100%;
    height: 100vh;
    margin: 0 auto;
    padding: 29px 20px 20px;

```

File: src/styles/blocks/menu.scss [hunk 20]
```diff
@@ -0,0 +1,116 @@
...
+
+  &__exit {
+    background-image: url(../images/photos/exit.svg);
+  }
+
+  &__list {
+    margin-bottom: 40px;
+    padding: 0;
+
+    max-width: 1020px;
```
Comment (SerhiiKirik) [inline src/styles/blocks/menu.scss]:
```suggestion
  &__list {
    max-width: 1020px;
    margin-bottom: 40px;
    padding: 0;
```

File: src/styles/blocks/menu.scss [hunk 21]
```diff
@@ -0,0 +1,116 @@
...
+    font-weight: 600;
+    font-size: 22px;
+    line-height: 22px;
+
+    letter-spacing: 2px;
+    text-transform: uppercase;
+    text-decoration: none;
+    transition-duration: 0.3s;
+
+    color: #1d1d1d;
```
Comment (SerhiiKirik) [inline src/styles/blocks/menu.scss]:
```suggestion
    color: $variables;

```

File: src/styles/blocks/menu.scss [hunk 22]
```diff
@@ -0,0 +1,116 @@
...
+
+  &__link {
+    font-weight: 600;
+    font-size: 22px;
+    line-height: 22px;
+
+    letter-spacing: 2px;
+    text-transform: uppercase;
+    text-decoration: none;
+    transition-duration: 0.3s;
```
Comment (SerhiiKirik) [inline src/styles/blocks/menu.scss]: the same

File: src/styles/blocks/page.scss [hunk 23]
```diff
@@ -0,0 +1,43 @@
+.page {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  color: $color-white;
+  scroll-behavior: smooth;
```
Comment (SerhiiKirik) [inline src/styles/blocks/page.scss]:
```suggestion
  color: $color-white;
  
  scroll-behavior: smooth;

```

File: src/styles/blocks/page.scss [hunk 24]
```diff
@@ -0,0 +1,43 @@
+.page {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  color: $color-white;
+  scroll-behavior: smooth;
+
+  &__body {
+    background-color: #1d1d1d;
```
Comment (SerhiiKirik) [inline src/styles/blocks/page.scss]:
```suggestion
    background-color: $variables;

```

File: src/styles/blocks/place.scss [hunk 25]
```diff
@@ -0,0 +1,55 @@
+.place {
+  &__photo-container {
+    position: relative;
+    width: 100%;
+    max-height: 256px;
+    overflow: hidden;
+
+    border-radius: 8px;
```
Comment (SerhiiKirik) [inline src/styles/blocks/place.scss]: split it

File: src/styles/blocks/place.scss [hunk 26]
```diff
@@ -0,0 +1,55 @@
...
+    width: 100%;
+    max-height: 256px;
+    overflow: hidden;
+
+    border-radius: 8px;
+
+    &::after {
+      content: "";
+      display: block;
+      padding-bottom: 100%;
```
Comment (SerhiiKirik) [inline src/styles/blocks/place.scss]: split

File: src/styles/blocks/place.scss [hunk 27]
```diff
@@ -0,0 +1,55 @@
...
+    }
+
+    transition: 0.3s;
```
Comment (SerhiiKirik) [inline src/styles/blocks/place.scss]: just transition?

File: src/styles/blocks/place.scss [hunk 28]
```diff
@@ -0,0 +1,55 @@
...
+
+    object-fit: cover;
+    object-position: center;
+  }
+
+  &__title {
+    margin: 0 auto 8px;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 140%;
```
Comment (SerhiiKirik) [inline src/styles/blocks/place.scss]:
```suggestion
    margin: 0 auto 8px;
    
    font-weight: 700;
    font-size: 24px;
    line-height: 140%;
```

File: src/styles/utils/_mixins.scss [hunk 29]
```diff
@@ -1,6 +1,37 @@
...
+    content: "";
+    display: block;
+    height: 1px;
+    border-radius: 8px;
+
+    background-color: #{$color};
+    transform: scale(0);
+
+    transition: scale, 0.3s;
+    pointer-events: none;
```
Comment (SerhiiKirik) [inline src/styles/utils/_mixins.scss]: sort it out

**Review by SerhiiKirik — APPROVED** [review]
Well done 🔥  ( It's reject )0
Let's fix some problems

File: src/styles/blocks/header.scss [hunk 30]
```diff
@@ -0,0 +1,105 @@
...
+    background-image: url(../images/photos/phone.svg);
+
+    @include onTablet {
+      &::before {
+        content: "+1 234 5555-55-55";
+        position: relative;
+        top: 50%;
+
+        transform: translate(-100%, -50%);
+        transition-duration: 0.3s;
```
Comment (SerhiiKirik) [inline src/styles/blocks/header.scss]: r
```suggestion
        transition: opacity 0.3s;
```

File: src/styles/blocks/about.scss (hunk 4 above)
Comment (SerhiiKirik) [reply src/styles/blocks/about.scss]: ???????????????

File: src/styles/utils/_typography.scss [hunk 31]
```diff
@@ -0,0 +1,32 @@
...
+  font-family: Poppins, sans-serif;
+  font-weight: 700;
+}
+
+%t-poppins-32-medium {
+  @extend %t-poppins-medium;
+  font-size: 32px;
+  line-height: 32px;
+
+  letter-spacing: -1.5px;
```
Comment (SerhiiKirik) [inline src/styles/utils/_typography.scss]:
```suggestion
```

File: src/styles/blocks/button.scss [hunk 32]
```diff
@@ -0,0 +1,39 @@
...
+  line-height: 27px;
+  color: $color-dark-black;
+
+  background: $color-white;
+  border-radius: 40px;
+
+  text-decoration: none;
+
+  cursor: pointer;
+  transition: all 0.3s;
```
Comment (SerhiiKirik) [inline src/styles/blocks/button.scss]: transition-duration: 0.3 - the same
use only param that you use

File: src/styles/blocks/card.scss [hunk 33]
```diff
@@ -0,0 +1,51 @@
...
+  margin: 0 auto 24px;
+  padding: 10px;
+
+  gap: 24px;
+
+  border: 1px solid transparent;
+  border-radius: 10px;
+  box-shadow: 1px 4px 16px rgba(232, 232, 240, 0.4);
+
+  transition-duration: 0.3s;
```
Comment (SerhiiKirik) [inline src/styles/blocks/card.scss]:
```suggestion
  grid-row: 1;

  max-width: 324px;
  margin: 0 auto 24px;
  padding: 10px;

  border: 1px solid transparent;
  border-radius: 10px;
  background-color: $color-dark-black;
  box-shadow: 1px 4px 16px rgba(232, 232, 240, 0.4);

  transition-duration: 0.3s;

```

Comment (SerhiiKirik) [reply src/styles/blocks/card.scss]: transition-duration: 0.3s; ???

File: src/styles/blocks/card.scss [hunk 34]
```diff
@@ -0,0 +1,51 @@
...
+
+  &__images {
+    margin-bottom: 32px;
+    width: 100%;
+  }
+
+  &__title {
+    margin: 0 auto 8px;
+
+    @extend %t-poppins-24-bold;
```
Comment (SerhiiKirik) [inline src/styles/blocks/card.scss]:
```suggestion
    @extend %t-poppins-24-bold;
    
    margin: 0 auto 8px;
```

Comment (SerhiiKirik) [reply src/styles/blocks/card.scss]: in ALL CASES!!!

File: src/styles/blocks/header.scss [hunk 35]
```diff
@@ -0,0 +1,105 @@
...
+
+        display: block;
+        width: max-content;
+        padding-right: 20px;
+        font-weight: 600;
+        font-size: 16px;
+        line-height: 27px;
+        letter-spacing: 1.5px;
+        color: $color-white;
```
Comment (SerhiiKirik) [inline src/styles/blocks/header.scss]: split this

File: src/styles/blocks/header.scss [hunk 36]
```diff
@@ -0,0 +1,105 @@
...
+        transform: translate(-100%, -50%);
+        transition-duration: 0.3s;
+
+        display: block;
+        width: max-content;
+        padding-right: 20px;
+        font-weight: 600;
+        font-size: 16px;
+        line-height: 27px;
+        letter-spacing: 1.5px;
```
Comment (SerhiiKirik) [inline src/styles/blocks/header.scss:66]: 1.5 or -1.5?
```suggestion
        letter-spacing: 1.5px;

```

File: src/styles/blocks/menu.scss [hunk 37]
```diff
@@ -0,0 +1,117 @@
...
+    display: block;
+
+    width: max-content;
+
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 27px;
+    color: $color-dark-black;
+
+    margin-bottom: 16px;
```
Comment (SerhiiKirik) [inline src/styles/blocks/menu.scss]:
```suggestion
    width: max-content;
    margin-bottom: 16px;

    font-weight: 600;
    font-size: 16px;
    line-height: 27px;
    color: $color-dark-black;
```

File: src/styles/blocks/menu.scss [hunk 38]
```diff
@@ -0,0 +1,117 @@
...
+      font-weight: 700;
+    }
+
+    @include link-after-hover($color-black);
+
+  }
+
+  &__click {
+    display: inline-block;
+    padding: 0 0 8px;
```
Comment (SerhiiKirik) [inline src/styles/blocks/menu.scss]:
```suggestion
    display: inline-block;
    
    padding: 0 0 8px;

```

File: src/styles/blocks/place.scss [hunk 39]
```diff
@@ -0,0 +1,58 @@
+.place {
+  &__photo-container {
+    position: relative;
+
+    overflow: hidden;
+
+    max-height: 256px;
+    width: 100%;
+    border-radius: 8px;
```
Comment (SerhiiKirik) [inline src/styles/blocks/place.scss]:
```suggestion
    width: 100%;
    
    border-radius: 8px;

```

File: src/styles/blocks/place.scss [hunk 40]
```diff
@@ -0,0 +1,58 @@
...
+
+    &::after {
+      content: "";
+
+      display: block;
+
+      padding-bottom: 100%;
+    }
+
+    transition-duration: 0.3s;
```
Comment (SerhiiKirik) [inline src/styles/blocks/place.scss]: 🤯

File: src/styles/utils/_mixins.scss [hunk 41]
```diff
@@ -1,6 +1,39 @@
...
+    display: block;
+
+    height: 2px;
+    border-radius: 8px;
+
+    background-color: #{$color};
+    transform: scale(0);
+
+    transition: scale, 0.3s;
+    pointer-events: none;
```
Comment (SerhiiKirik) [inline src/styles/utils/_mixins.scss]:
```suggestion
    height: 2px;
    
    border-radius: 8px;
    background-color: #{$color};
    
    transition: scale 0.3s;
    transform: scale(0);
    pointer-events: none;
```

File: src/index.html [hunk 42]
```diff
@@ -1,13 +1,390 @@
...
+
+      <section class="page__section compare" id="compare">
+        <h2 class="compare__title">
+          Compare bikes
+        </h2>
+
+        <div class="container">
+          <div class="grid grid__cards">
+            <div class="card grid__item-desktop--1-4"
+            >
```
Comment (SerhiiKirik) [inline src/index.html]:
```suggestion
            <div class="card grid__item-desktop--1-4">
```

File: src/index.html [hunk 43]
```diff
@@ -1,13 +1,390 @@
...
+                most comfortable ride in town.
+              </p>
+
+              <div class="card__price">
+                $ 2 590
+              </div>
+            </div>
+
+            <div class="card grid__item-desktop--9-12"
+            >
```
Comment (SerhiiKirik) [inline src/index.html]:
```suggestion
            <div class="card grid__item-desktop--9-12">
```

File: src/index.html [hunk 44]
```diff
@@ -1,13 +1,390 @@
...
+          </h2>
+
+          <div class="details__place place">
+            <div class="place__photos">
+              <div class="grid place__distance">
+                <div class="grid__item-tablet--1-4 grid__item-desktop--1-8">
+                  <div class="place__photo-container">
+                    <img src="./images/photos/photo1.jpg"
+                         alt="Photo 1"
+                         class="place__photo">
```
Comment (SerhiiKirik) [inline src/index.html]:
```suggestion
                    <img src="./images/photos/photo1.jpg"
                         alt="Photo 1"
                         class="place__photo"
                     >
```

File: src/index.html [hunk 45]
```diff
@@ -1,13 +1,390 @@
...
+                  </div>
+                </div>
+
+                <div class="grid__item-tablet--5-7 grid__item-desktop--9-12">
+                  <div class="place__photo-container">
+                    <img src="./images/photos/photo2.jpg"
+                         alt="Photo 2"
+                         class="place__photo">
```
Comment (SerhiiKirik) [inline src/index.html]:
```suggestion
                    <img src="./images/photos/photo2.jpg"
                         alt="Photo 2"
                         class="place__photo"
                     >
```

**Review by SerhiiKirik — CHANGES_REQUESTED** [review]
Well done, almost. let's fix some problem

File: src/styles/blocks/header.scss (hunk 36 above)
Comment (VolodymyrKirichenko) [reply src/styles/blocks/header.scss:66]: 1.5 (everything is correct here)

File: src/styles/blocks/button.scss [hunk 46]
```diff
@@ -0,0 +1,39 @@
...
+  line-height: 27px;
+  color: $color-dark-black;
+
+  background: $color-white;
+  border-radius: 40px;
+
+  text-decoration: none;
+
+  cursor: pointer;
+  transition-duration: 0.3s;
```
Comment (SerhiiKirik) [inline src/styles/blocks/button.scss]:
```suggestion
  transition: box-shadow 0.3s;

```

File: src/styles/blocks/button.scss [hunk 47]
```diff
@@ -0,0 +1,39 @@
+.button {
+  display: flex;
+
```
Comment (SerhiiKirik) [inline src/styles/blocks/button.scss]:
```suggestion
```

File: src/styles/blocks/card.scss [hunk 48]
```diff
@@ -0,0 +1,51 @@
...
+  padding: 10px;
+
+  gap: 24px;
+
+  border: 1px solid transparent;
+  border-radius: 10px;
+  background-color: $color-dark-black;
+  box-shadow: 1px 4px 16px rgba(232, 232, 240, 0.4);
+
+  transition-duration: 0.3s;
```
Comment (SerhiiKirik) [inline src/styles/blocks/card.scss]:
```suggestion
  transition: transform  0.3s;

```

File: src/styles/blocks/contact-us.scss [hunk 49]
```diff
@@ -0,0 +1,64 @@
...
+      line-height: 48px;
+    }
+
+    margin-bottom: 48px;
+
+    text-align: center;
+
+    letter-spacing: $spacing;
+  }
+
```
Comment (SerhiiKirik) [inline src/styles/blocks/contact-us.scss]:
```suggestion
    margin-bottom: 48px;

    text-align: center;

    letter-spacing: $spacing;
    
     @include onTablet {
        font-size: 48px;
        line-height: 48px;
    }
  }
```

File: src/styles/blocks/header.scss [hunk 50]
```diff
@@ -0,0 +1,110 @@
...
+  background-size: cover;
+  background-position: right;
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    height: 100vh;
+
```
Comment (SerhiiKirik) [inline src/styles/blocks/header.scss]:
```suggestion

```

File: src/styles/blocks/header.scss [hunk 51]
```diff
@@ -0,0 +1,110 @@
...
+    }
+  }
+
+  &__nav {
+    display: flex;
+    justify-content: space-between;
+
+    max-width: 1020px;
+    width: 100%;
+
```
Comment (SerhiiKirik) [inline src/styles/blocks/header.scss]:
```suggestion

```

File: src/styles/blocks/header.scss [hunk 52]
```diff
@@ -0,0 +1,110 @@
...
+        position: relative;
+        top: 50%;
+
+        transform: translate(-100%, -50%);
+        transition: opacity 0.3s;
+
+        display: block;
+
+        padding-right: 20px;
+
```
Comment (SerhiiKirik) [inline src/styles/blocks/header.scss]:
```suggestion

```

File: src/styles/blocks/header.scss (hunk 16 above)
Comment (SerhiiKirik) [reply src/styles/blocks/header.scss]: Do you ignore me?
```suggestion
    transition: background-image 0.3s;

```

File: src/styles/blocks/menu.scss [hunk 53]
```diff
@@ -0,0 +1,119 @@
...
+  overflow: auto;
+
+  background-color: $color-menu;
+
+  &__container {
+    max-width: 1260px;
+    width: 100%;
+
+    height: 100vh;
+
```
Comment (SerhiiKirik) [inline src/styles/blocks/menu.scss]:
```suggestion
    height: 100vh;
```

File: src/styles/blocks/menu.scss [hunk 54]
```diff
@@ -0,0 +1,119 @@
...
+    margin-bottom: 44px;
+  }
+
+  &__exit {
+    background-image: url(../images/photos/exit.svg);
+  }
+
+  &__list {
+    max-width: 1020px;
+
```
Comment (SerhiiKirik) [inline src/styles/blocks/menu.scss]:
```suggestion
```

File: src/styles/blocks/menu.scss [hunk 55]
```diff
@@ -0,0 +1,119 @@
...
+  &__link {
+    font-weight: 600;
+    font-size: 22px;
+    line-height: 22px;
+
+    letter-spacing: 2px;
+    text-transform: uppercase;
+    text-decoration: none;
+    transition-duration: 0.3s;
```
Comment (SerhiiKirik) [inline src/styles/blocks/menu.scss]: here you have to do by yourself

File: src/styles/blocks/menu.scss [hunk 56]
```diff
@@ -0,0 +1,119 @@
...
+
+    @include link-after-hover($color-black);
+
+  }
+
+  &__phone-number {
+    display: block;
+
+    width: max-content;
+
```
Comment (SerhiiKirik) [inline src/styles/blocks/menu.scss]:
```suggestion
```

File: src/styles/blocks/menu.scss [hunk 57]
```diff
@@ -0,0 +1,119 @@
...
+    text-transform: uppercase;
+    text-decoration: none;
+
+    @include onDesktop {
+      font-weight: 700;
+    }
+
+    @include link-after-hover($color-black);
+
```
Comment (SerhiiKirik) [inline src/styles/blocks/menu.scss]:
```suggestion
```

File: src/styles/utils/_mixins.scss [hunk 58]
```diff
@@ -1,6 +1,39 @@
...
+    content: "";
+
+    display: block;
+
+    height: 2px;
+
+    border-radius: 8px;
+    background-color: #{$color};
+
+    transition: scale, 0.3s;
```
Comment (SerhiiKirik) [inline src/styles/utils/_mixins.scss]:
```suggestion
    transition: transform 0.3s;
```

**Review by SerhiiKirik — CHANGES_REQUESTED** [review]
Good job 👍 
We are almost done with this PR
Check comments & fix problem 😉

**Pull request merged**